### PR TITLE
js assignment 1,2,3,4,5,6,9

### PR DESCRIPTION
**Pull request by harshalkotkar**

### CODE REVIEW

**Review by MKadeer17 — APPROVED** [review]
**Please Provide the Proper title** in number of html file.

File: HARSHAL/javascript/assignment1/assignment1(1).html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <label for="a">enter a:</label>
+    <input type="number" id="a" value="" /><br />
+    <label for="b">enter b:</label>
+    <input type="number" id="b" value="" /><br />
+
+    <h1>WITHOUT USING TEMP VARIABLE</h1>
+    <button onclick="myFunction()">click me</button>
+    <div id="st"></div>
+    <div id="nd"></div>
+
```
Comment (JafarPathan) [inline HARSHAL/javascript/assignment1/assignment1(1).html:19]: remove extra line

File: HARSHAL/javascript/assignment1/assignment1(1).html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <h1>USING TEMP VARIABLE</h1>
+    <button onclick="myFunction1()">click me</button>
+    <div id="one"></div>
+    <div id="two"></div>
+
```
Comment (JafarPathan) [inline HARSHAL/javascript/assignment1/assignment1(1).html:24]: remove extra line

**Review by JafarPathan — COMMENTED** [review]
FOR All File 
1. Do proper indentation (use 4 spaces for indentation)
2. remove unwanted lines

**Comment by harshalkotkar** [comment]
> FOR All File
> 
> 1. Do proper indentation (use 4 spaces for indentation)
> 2. remove unwanted lines

done Sir

**Review by saifsyed566 — APPROVED** [review]

**Pull request merged**